examples/simple_hpc_pbs_zfs/test.yml

```diff
@@ -0,0 +1,5 @@
+steps:  
```

Comment:
Is there any tests to be done specifically for ZFS that can be added there ?

Reply:
I've added a test to check there is a ZFS pool online

Reply:
I've added a test to make sure there is at least one ZFS pool online